.github/ISSUE_TEMPLATE/customer-issue.md

```diff
@@ -0,0 +1,37 @@
+---
+name: Customer Issue
+about: Create a report to help us improve
+title: '[CRI]'
+labels: kind/bug, kind/customer-issue
+assignees: ''
+
+---
+
+ **Describe the issue**
+A clear and concise description of what the issue is.
+
+ **To Reproduce**
+Steps to reproduce the behavior:
+1. 
+2.
+3.
+
+ **Expected behavior**
+A clear and concise description of what you expected to happen.
+
+ **Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+ **Knative version**
+
+ **Customer Info**
+Name of the customer, the use case trying to solve, and any more information about the customer that is relevant to this issue.
```

Comment:
this is confusing, I don't think many people will be making issues on behalf of a customer.

Reply:
I'm not sure that we need to require a customer identifier here.

I suggested that we may want to track customer issues separately from development-filed bugs, as the former has at least one external user waiting on a fix or clarification, while the latter are tracking and hygiene issues but may not have the same criticality.

Reply:
Thinking more about it, this will may result in privacy violations too.
For now I'll remove this particular template. We can explicitly add  the customer-issue label for relevant bugs.